apis/mlops/agent/agent.proto

```diff
@@ -27,8 +27,35 @@ message ModelEventMessage {
   Event event = 5;
   string message = 6;
   uint64 availableMemoryBytes = 7;
+  ModelConfig modelConfig = 8;
 }
 
+message ModelConfig {
+  enum Type {
+        UNKNOWN_TYPE = 0;
+        MLSERVER = 1;
+        TRITON = 2;
+        }
+
+  Type type = 1;
```

Comment:
Do we still need the type if you're using oneof? We could just test which of the oneof configs is present via doing a switch on the config type. This would also avoid invalid messages with one Type and the incompatible config.

Reply:
Removed it.